core/src/main/java/com/netflix/conductor/core/execution/tasks/SystemTaskWorker.java

```diff
@@ -118,7 +119,11 @@ void pollAndExecute(WorkflowSystemTask systemTask, String queueName) {
             }
             LOGGER.debug("Polling queue: {} with {} slots acquired", queueName, acquiredSlots);
 
-            List<String> polledTaskIds = queueDAO.pop(queueName, acquiredSlots, 200);
+            long taskStartMillis = Instant.now().toEpochMilli();
+
+            List<String> polledTaskIds = queueDAO.pop(queueName, acquiredSlots, 0);
```

Comment:
The change from 200 -> 0 (long poll delay) is an attempt to increase polling responsiveness of system tasks and additional code to monitor the time taken for polling. I've found it to be bottlenecking at this point so I might revert the change.